www/src/Gallery.jsx

```diff
@@ -10,28 +10,50 @@ import { mockBlockIdByType } from '@edx/frontend-lib-content-components/editors/
 
 export const EditorGallery = () => {
   const [blockType, setBlockType] = React.useState('html');
+  const [mockRaw, setMockRaw] = React.useState(false);
+
   const blockIds = Object.keys(blockTypes).reduce((obj, blockTypeKey) => {
     const type = blockTypes[blockTypeKey];
     return { ...obj, [type]: mockBlockIdByType(type) };
   }, {});
```

Comment:
```suggestion
  }, {});
  
  // and add a blockid here that contians whay you want and is of the blocktype HTML
```

Reply:
you might need to make one more level of abstraction here (keep a list of all the views we want to present, not just block types, but you catch my drift)